### ISSUE TITLE
0.20.0 aarch64 wheels on PyPi seem to be bad

### ISSUE THREAD

**Issue by Polsaker** [kevinheavey/solders#86]
Hello, testing this lib on an ARM machine I'm getting the following error (this is not happening on an x86 container):

```py
  File "/usr/local/lib/python3.12/site-packages/solana/rpc/api.py", line 11, in <module>
    from solders.rpc.responses import (
ModuleNotFoundError: No module named 'solders.rpc.responses'
```

Checking the wheel files on PyPI you can notice that the aarch64 wheels are suspiciously smaller than the wheels for other archs:

![image](https://github.com/kevinheavey/solders/assets/2000719/ec0ee041-a5bb-417a-8034-ddd772b13657)

**Comment by kevinheavey** [comment]
Unfortunately this is deliberate because the required dependencies (specifically `ring`) don't compile on linux aarch64, so we leave them out:

https://github.com/kevinheavey/solders/blob/b14a24010b1dd2e99fb5011e4e4503d3789a1e9c/.github/workflows/build.yml#L248C16-L248C59

I just triggered a CI run that tries to include the `ring`  dependency so you can see https://github.com/kevinheavey/solders/actions/runs/8264519529/job/22608292459?pr=87

Even more annoying is that we don't even need that code, but a bunch of Solana crates depend on `reqwest` when they really shouldn't. For example [solana-account-decoder](https://github.com/anza-xyz/agave/blob/master/account-decoder/Cargo.toml) depends on `solana-config-program` which indirectly depends on `reqwest` :(

**Comment by Polsaker** [comment]
Maybe it would be better not to offer aarch64 wheels then? So pip install can at least attempt to build locally by default.

For us forcing pip to build locally worked just fine (it just added a few minutes to all our pipelines 🙃)

**Comment by kevinheavey** [comment]
Oh that's weird that it compiles for you on Linux aarch64 right? Since it doesn't compile in CI

**Comment by kevinheavey** [comment]
Thanks for the info, that led me to discover there actually was a way to get `ring` to compile in CI. Fixed by #87
